src/trousse/operations_list.py

```diff
@@ -0,0 +1,184 @@
+import collections
+import typing
+from typing import List, Union, Any
```

Comment:
About the LGTM alerts, we want to keep both of these imports to make line 5 work right?

Reply:
well, yes